docs/docs/binary-markets.md

```diff
@@ -33,14 +33,15 @@ Uniswap's CPMM and a suggestion from Manifold user Pepe.
       - If you bet M$100 on NO and the resulting probability without fees would be 10%, then you pay M$100 _ 13% _ 10% = M$1.3.
       - If you bet M$100 on YES and the resulting probability without fees would be 90%, then you pay `M$100 * 13% * 10% = M$1.3`.
     - The fees are used to provide a commission to the market creator and to subsidize trading within the market.
+    - 6% goes to the market creator, 6% to liquidity providers, and 1% is burnt to prevent inflation
```

Comment:
This actually changed since the PR was created: https://github.com/manifoldmarkets/manifold/commit/67abb942a09104fc529a20bef9aa1449fe5210e3